docs/writing-rules/testing-rules.md

```diff
@@ -69,82 +91,115 @@ eval(safe_get_user_input())
 Run the tests with the following:
 
 ```sh
-python -m semgrep --quiet --test rules/
+semgrep --test rules/
 ```
 
 Which will produce the following output:
 ```sh
-1 yaml files tested
-check id scoring:
---------------------------------------------------------------------------------
-(TODO: 2) rules/detect-eval.yaml
-	✖ insecure-eval-use                                  TP: 1 TN: 2 FP: 1 FN: 1
-	test: rules/detect-eval.py, expected lines: [5, 12], reported lines: [5, 15]
---------------------------------------------------------------------------------
-final confusion matrix: TP: 1 TN: 2 FP: 1 FN: 1
---------------------------------------------------------------------------------
+1/1: ✓ All tests passed
+No tests for fixes found.
 ```
 
-- True positives (`TP`) correspond to `ruleid`
-- True negatives (`TN`) correspond to `ok`
-- False positives (`FP`) correspond to `todook`
-- False negatives (`FN`) correspond to `todoruleid`
-
+As you can see, semgrep tests automatically avoid failing on lines marked with `# todoruleid` or `# todook`.
 To avoid failing on TODOs you can specify `--test-ignore-todo`:
 
```

Comment:
## Storing rules and test targets in different directories

Creating different directories for rules and tests helps users manage a growing library of custom rules.